.vsts-dotnet-ci.yml

```diff
@@ -5,64 +5,70 @@ trigger:
 - vs*
 
 jobs:
-- job: BootstrapMSBuildOnFullFrameworkWindows
```

Comment:
I'm confused by why all these other changes. Like why did you move them around? And did you just move them around? It's hard to tell what the other changes are.

Reply:
The changes to this file should be viewed by commits. I have a commit where all I did was move the only non-bootstrap build to the top so that it runs first.

Reply:
Noting that I reverted this change. My goal was to change the order these jobs run in, but they're listed in alphabetical order anyway 😕